modules/battery_manager/README.md

```diff
@@ -0,0 +1,147 @@
+# Battery Manager
+
+### LEVEL easy:
```

Comment:
I don't understand why this is the first thing in the README with no preamble to explain what it's talking about. It's going to confuse readers.

Reply:
I think this is maybe a remnant from the original test plan written for this module.

Reply:
Yeah, I'm not really sure since I didn't write the readme. I'll try and make it resemble the other readmes.